meteor/client/helpers/google_maps.js

```diff
@@ -94,7 +105,7 @@ gmaps = {
 
       var googleLoc = new google.maps.LatLng(latLng.lat, latLng.lng);
       gmaps.selfMarker.setPosition(googleLoc);
-    
+
     } else {
 
       if (navigator.geolocation) navigator.geolocation.getCurrentPosition(function(pos) {

```

Comment:
@drewproud should the if statement here have brackets around it? This looks like it might be a copy and paste error...